### DISCUSSION TITLE
Is this library supports low latency streaming?

### DISCUSSION THREAD

**Discussion by pjcondenast20** [HaishinKit/HaishinKit.swift#1631] [Q&A]
Hello, I'm trying to achieve low latency streaming, ideally with a glass-to-glass latency of <500ms. Does this library support that?

If so, does it support all the available protocols or any specific protocol? Pls assist.

I have now tried streaming over an RTMP protocol to my local server and see about 2.5 seconds of latency.

**Comment by shogo4405** [comment]
The only supported protocols are RTMP and SRT.
